app/controllers/api/v1/capes_controller.rb

```diff
@@ -8,18 +9,27 @@ def show
     render json: Cape.find(params[:id])
   end
 
- def create
-  cape = Cape.new(cape_params)
-    if cape.save
-     render json: cape
-    else
-     render json: { errors: cape.errors.full_messages }
+  def create
+    if user_signed_in?
```

Comment:
The whole reason you defined `authorize` user was to get rid of line 13 (even if you didnt notice.) now that line 13 is there, the user signin check can be removed from your methods.